changelog/unreleased/kong/bump-lua-resty-openssl.yml

```diff
@@ -1,2 +1,2 @@
-message: "Bumped lua-resty-openssl from 1.4.0 to 1.5.0"
+message: "Bumped lua-resty-openssl from 1.4.0 to 1.5.1"
```

Comment:
We'd better state why this bump is required.

Reply:
There is nothing worth to mention about that bump other than that we are now using the lastest, see:
https://github.com/fffonion/lua-resty-openssl/blob/master/CHANGELOG.md
https://github.com/fffonion/lua-resty-openssl/compare/1.5.0...1.5.1

Reply:
I added the reason.